src/renderer/components/toolbars/MinerSummary.tsx

```diff
@@ -0,0 +1,42 @@
+import { Box, Typography } from '@mui/material';
+
+import { Separator } from '../Separator';
+import { useObservableState, useMinerActive } from '../../hooks';
+import { minerState$ } from '../../../models';
+import * as formatter from '../../services/Formatters';
+import { minerStatistics$ } from '../../services/StatisticsAggregator';
+
+export function MinerSummary() {
+  const [minerState] = useObservableState(minerState$, null);
+  const [minerStatistic] = useObservableState(minerStatistics$, null);
+  const minerActive = useMinerActive();
+
+  const items = [
+    { title: 'Coin', content: minerState?.currentCoin },
+    { title: 'Hashrate', content: formatter.hashrate(minerStatistic?.hashrate) },
+  ];
+
+  return (
+    <Box>
+      {minerActive && (
+        <Typography sx={{ mr: 2 }}>
+          {items
+            .map((item) => (
+              <>
+                <strong>{item.title}</strong>: {item.content}
+              </>
+            ))
+            .reduce((acc, x) =>
+              acc === null ? (
+                x
+              ) : (
+                <>
+                  {acc} <Separator /> {x}
+                </>
+              )
+            )}
```

Comment:
Consider using `Array#join` here instead of `Array#reduce` with a ternary.

Reply:
This was actually the first thing I tried.  The problem is that `Array#join` expects a string separator.  If I try passing in `'|'` instead of `<Separator />` then the method stringifies all of the elements.

Using `reduce()` wasn't my first choice, but after reading https://stackoverflow.com/questions/34034038/how-to-render-react-components-by-using-map-and-join it looks like that's the only way to do this.